share/external/fmu_packages_installation_guide.md

```diff
@@ -0,0 +1,229 @@
+
+- To install another version of RMS, simply replace references to «14.2.2» with the RMS
+  version of your choice
+  - Note that RMS license is critical to be able to run RMS, 
+    but about the RMS license, please contact AspenTech for details, help and for getting support assistance. 
+
+## Install FMU components for RMS
+
+Create this file from the bash-script-lines below :
+(assuming that rms command is /opt/roxar/rms/rms)
+
+$HOME/venv/tutor/bin/run_external
+
+```sh
+#!/usr/bin/env bash  
+# wrapper script so running system python from RMS will work  
+unset PYTHONPATH  
+unset PYTHONUSERBASE  
+unset LD_LIBRARY_PATH  
+unset LD_LIBRARY64_PATH  
+
+exec $@  
+```  
+
+Then copy the file here like this:
+
+```sh
+cp $HOME/venv/tutor/bin/run_external /opt/roxar/rms/run_external 
+```
+
+## Modify RMS startup
+
+In /opt/roxar/rms * folder: rename “rms” to “rms_orig”, then make a new rms script containing this:
+
+(*assume that rms command is /opt/roxar/rms/rms)
+
+```sh
+#!/bin/sh
+
+export PATH=/opt/roxar/rms:$PATH
+
+/opt/roxar/rms/rms_orig $@
+```
+
+## Install Python packages for RMS
+
+In this example, the RMS Python packages will be installed automatically under $HOME/.roxar/python. There are better strategies if multiple users shall share the same libraries.
+
+- Run roxenv script which is in the RMS installation folder tree, to activate RMS’s python:
+
+```
+/opt/roxar/rms/versions/14.2.2/bin/LINUX_64/roxenv
+which python
+/opt/roxar/rms/versions/14.2.2/linux-amd64-gcc_4_8-release/bin/python
+```
+
+- Now, given RMS’s python is active:
+
+  - Install roxar_api_utils from https://github.com/RoxarAPI/roxar_api_utils in the user folder
+
+  - Run these installation commands:
+
+```
+python -m pip install pip -U --user
+python -m pip install xtgeo --user
+python -m pip install wheel --user
+python -m pip install setuptools_scm --user
+python -m pip install fmu-config --user
+```  
+
+## Run RMS for the first time
+
+Run RMS for the virtual “tutor” environment
+
+Now go to the folder “model” where the RMS project is present and type: 
+
+```
+rms
+```
+
+Run the workflow MAIN and see if it runs without issues!
+
+
+       
+
+# Optional installation of the additional FMU-tools Resinsight and Webviz for visualization of results
+
+## Resinsight installation
+
+- Resinsight is an Open Source tool that can be used to visualize results from Reservoir Simulation.
+
+- Here is the Getting Started information for download and install Resinsight on Linux:
+
+https://resinsight.org/getting-started/download-and-install/linux-installation/
+
+
+##  WEBVIZ for Subsurface
+
+Webviz-subsurface is an Open Source tool to visualize Subsurface data via a web-based user interface.
+
+- Here is a demo of Webviz:
+
+https://webviz-subsurface-example.azurewebsites.net
+
+- See the GITHUB-repository for Webviz here:
+
+https://github.com/equinor/webviz-subsurface
+
+- Installation instruction:
```

Comment:
Maybe we can remove the installation part for webviz since that is covered in the webviz repo we link to?